tests/test_range.py

```diff
@@ -205,11 +205,11 @@ def test_implicit_date_change(self):
                           datetime(2017, 6, 17, 0, 0, 0))
 
         self.assert_range('11:59pm on Feb 28',
-                          datetime(2017, 2, 28, 23, 59, 0),
-                          datetime(2017, 3, 1, 0, 0, 0))
+                          datetime(2018, 2, 28, 23, 59, 0),
```

Comment:
why not use freeze gun for all these tests?

Reply:
Looks like freeze_time is being used on line 11 already

Reply:
From what I can tell, this means that every time the test or the code makes a call to get the system time, it would get the next time specified in the range. If so, the code would get a different code than the test. It also means the result depends on how many times the code calls `datetime.now()`, which would make the test brittle. Is there any advantage to this kind of range instead of having just one test time?

Reply:
oh i am sorry i didn't notice that it is being used already. Also there is no problem on freezing on one time.